inscriptions/ISic004431.xml

```diff
@@ -0,0 +1,195 @@
+                    <msContents>
+                        <textLang mainLang="grc">Ancient Greek</textLang>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc>
+                            <supportDesc>
+                                <support>
+                                    <material ana="#material.stone.limestone" ref="http://www.eagle-network.eu/voc/material/lod/66.html">limestone</material>
+                                    <objectType ana="#object.base" ref="http://www.eagle-network.eu/voc/objtyp/lod/57.html">base</objectType>
+                                    <dimensions>
+                                        <height unit="cm">65</height>
+                                        <width unit="cm">65</width>
+                                        <depth unit="cm"/>
+                                    </dimensions>
+                                </support>
+                                <condition ana="#condition.complete"/>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout>
+                                    <rs ana="#execution.chiselled" ref="http://www.eagle-network.eu/voc/writing/lod/1">chiselled</rs>
+                                    <damage ana="text_condition.complete"/>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote><!--Default value and requires editing-->
+                                <locus from="line1" to="line1">Line 1</locus>
+                                <dimensions type="letterHeight">
+                                    <height unit="mm"/>
+                                </dimensions>
+                                <locus from="line1" to="line2">Interlineation line 1 to 2</locus>
+                                <dimensions type="interlinear">
+                                    <height unit="mm"/>
+                                </dimensions>
+                            </handNote>
+                        </handDesc>
+                    </physDesc>
+                    <history>
+                        <origin>
+                            <origPlace>
+                                <placeName type="ancient" ref="http://pleiades.stoa.org/places/462527">Tyndaris</placeName>		
+                                <placeName type="modern" ref="http://sws.geonames.org/10282151">Tindari</placeName>
+                                <geo>38.14357, 15.04308</geo>
+                            </origPlace>
+                            <origDate datingMethod="#julian" notBefore-custom="-0300" notAfter-custom="-0001">Hellenistic</origDate>
+                        </origin>
+                        <provenance type="found">Found at Contrada San Leo in the municipality of Oliveri, at the foot of Tyndaris</provenance>
```

Comment:
this needs `<provenance type="found" subtype="discovered" when="1970">` and I would insert `<geo cert="medium">38.132401, 15.052095</geo><!--approx location marker pending identification of villa site-->` inside the provenance elemnt at the end of your text (i.e. after the word Tyndaris)